WL-3743 Added 'search Users' textbox in Site Info.

ouit0408 opened the pull request:
Added Search input and clear search in the chef_site-siteInfo-list.vm.
If search word is in session state then Participant List is filtered for
the search term in each Participant's displayName and displayId.
Corrected the position of search box in the participant's table.
This pull request is linked to https://github.com/ox-it/wl-reference/pull/125

buckett commented on site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java:
```diff
@@ -4668,6 +4669,35 @@ public void doSite_search_clear(RunData data, Context context) {
+				.getPortletSessionState(((JetspeedRunData) data).getJs_peid());
+
+		// read the search form field into the state object
+		String search = StringUtils.trimToNull(data.getParameters().getString(
```
I think this needs escaping for XSS filtering, otherwise someone can inject javascript into a WebLearn URL.

buckett replied:
Doesn't this need XSS filtering?

ouit0408 replied:
Added Validator.escapeHtml for search string.

ouit0408 committed:
WL-3743 Added 'search Users' textbox in Site Info.

Added Search input and clear search in the chef_site-siteInfo-list.vm.
If search word is in session state then Participant List is filtered for
the search term in each Participant's displayName and displayId.
Corrected the position of search box in the participant's table
